operating/view-dashboard.adoc

```diff
@@ -6,58 +7,100 @@ include::modules/common-attributes.adoc[]
+* To view data for all clusters, select *All clusters*. 
+* To view data for specific namespaces, select them from the *Resources* drop-down list. Note that all namespaces are automatically included when you select *All clusters*. 
+
+[id="policy-violations-by-severity"]
+== Policy violations by severity
+
+The *Policy violations by severity* widget displays the total number of violations and displays the number of violations by severity level. Severity levels are color-coded with gray for low severity; yellow for medium severity; orange for high severity; and red for critical severity. 
+
+You can select items to view additional information as follows:
+
+* To view all violations of a specific severity, click the count for the desired severity level. 
+* To view all violations of any severity, select *View all*. Choosing a violations category or *View all* displays the *Violations* page and the menu bar reflects any *Cluster* or *Namespace* selections that you made in the dashboard.
+
+The *Policy violations by severity* widget also displays the three most recent violations that have a severity level of *Critical*. Choosing a specific violation displays the xref:../operating/respond-to-violations.adoc#view-violation-details[Violations detail] page that provides additional information about the violation.
+
+[id="images-at-most-risk"]
+== Images at most risk
+
+The *Images at most risk* widget lists images that have the highest risk score. Risk score is determined by an analysis that uses number of critical common vulnerabilities and exposures (CVEs), severity of CVEs, and other factors.
+
+You can configure items for display in the widget by selecting the following options:
+
+* *Image vulnerabilities*:
+**  *Fixable CVEs*: Displays the number of fixable CVEs in the *Critical CVEs* and *Important CVEs* columns.
+**  *All CVEs*: Displays the total number of CVEs in the *Critical CVEs* and *Important CVEs* columns.
+* *Image status*: 
+** *Active images*: Displays the active images at most risk. 
+** *All images*: Displays all images at most risk.
 
-[id="violations-by-cluster"]
-== Violations by Cluster
+You can also perform the following actions in this widget:
 
-The *Violations by Cluster* widget shows a bar chart based on the number of violations in each cluster in different colors based on their severity.
+* To view CVEs for an image, click on the image name.
+* To view a list of all riskiest images, select *View all*.
 
-[id="top-risky-deployments"]
-== Top Risky Deployments
+[id="deployments-at-most-risk"]
+== Deployments at most risk
 
-The *Top Risky Deployments* widget displays the top five deployments at risk in your containerized environment.
+The *Deployments at most risk* widget lists deployments that have the highest risk, as determined by the risk priority. The widget displays the deployment name, cluster and namespace, and risk priority. 
```

Comment:
We talk about risk priority here but we talk about risk score in the context of images (see above). Why so? Intentional? Should we talk about risk priority everywhere since we don't really expose risk score?